tests/integration/testbase.py

```diff
@@ -5,8 +5,19 @@
 
 import packit.distgit
 import packit.upstream
-from ogr import GithubService
-from ogr.persistent_storage import PersistentObjectStorage
+
+try:
```

Comment:
why there are two try blocks? Is it possible to merge them together?

Reply:
I actually wasn't sure which one corresponds to which version: my changes are really just an educated guess.

I wouldn't actually bother merging the try/catch blocks and would instead drop ogr<0.5 code once 0.5 is in stable since the new packit won't work with old ogr anyway - this is really just a patch to have more greener CI.